index.html

```diff
@@ -157,6 +153,26 @@ <h3>Profesional Art<br> Printing Data</h3>
                 </li>                                
             </ul>
         </div>
-    </section>   
+    </section>
+    <footer>
+        <section class="contact-info">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="/login" method="post">
+                <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                <textarea id="msg" name="user_message" cols="30" rows="10" maxlength="500" required>Enter text here</textarea>
```

Comment:
- [ ] The form is still sending an empty message and that is bad because no one want to receive an empty message, I do see the required but it's not working.

Reply:
the issue of contact form fixed so please check it again

Reply:
> * [ ]  The form is still sending an empty message and that is bad because no one want to receive an empty message, I do see the required but it's not working.

the issue of contact form fixed so please check it again